packages/components/psammead-grid/src/index.jsx

```diff
@@ -97,7 +97,7 @@ const gridMediaQueries = ({
       min: groups[group].min,
       max: groups[group].max,
       styles: `
-      width: initial;
+      width: 100%;
```

Comment:
Block elements default to a width of 100%.
1. I'm curious why this rule is required.
2. What happens if it is removed altogether?

Reply:
> I'm curious why this rule is required and how changing it fixes this.
I don't really know tbh 😞 , please feel free to try and work this out. Fwiw, it is doing what MDN says it should do rather than rely on less clear, default behaviour.

> What happens if it is removed altogether?
I can give this a try, I'll see what happens

Reply:
Dropping `width: initial` seems to adjust to width to be narrower, not really sure why:
![image](https://user-images.githubusercontent.com/9645462/101481462-20c55380-394d-11eb-9f2b-f9c19f86af91.png)

Reply:
Hey so this is because after dropping the `width: initial`, it then inherits from another grid style, which is assigning the width to 66.66666%

![Screenshot 2020-12-09 at 13 21 00](https://user-images.githubusercontent.com/9042672/101635096-6fdcb880-3a21-11eb-8aa4-2bf5e628fa32.png)